commands/kubernetes.go

```diff
@@ -1017,14 +1017,14 @@ func mergeKubeconfig(clusterID string, remote, local *clientcmdapi.Config) error
 	remoteCtx, ok := remote.Contexts[remote.CurrentContext]
 	if !ok {
 		// this is a bug in the backend, we received incomplete/non-sensical data
-		return fmt.Errorf("the remote config has no context entry named %q. This is a bug, please open a ticket with DigitalOcean!",
+		return fmt.Errorf("the remote config has no context entry named %q -- this is a bug, please open a ticket with DigitalOcean",
 			remote.CurrentContext,
 		)
 	}
 	remoteCluster, ok := remote.Clusters[remoteCtx.Cluster]
 	if !ok {
 		// this is a bug in the backend, we received incomplete/non-sensical data
-		return fmt.Errorf("the remote config has no cluster entry named %q. This is a bug, please open a ticket with DigitalOcean!",
+		return fmt.Errorf("the remote config has no cluster entry named %q -- This is a bug, please open a ticket with DigitalOcean",
```

Comment:
`this` is lowercase here, but uppercase in the other occurrences

Reply:
Great catch Bouke -- fixed.